saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/suggester.js

```diff
@@ -133,10 +133,11 @@ var Suggester = (function() {
     	var fieldId = $field.attr("id"),
     		fieldValue = $field.val(),
     		$textArea = $("<textarea></textarea>"),
-    		$wrapper = $("<div></div>");
+    		$wrapper = $("<div></div>")
```

Comment:
I don't understand why sometimes you have a variable with $ at the beginning and sometimes without?